.github/workflows/release_wheel_creation.yml

```diff
@@ -0,0 +1,106 @@
+name: Pyomo Release Distribution Creation
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  manylinux:
+    name: ${{ matrix.TARGET }}/wheel_creation
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        include:
+        - os: ubuntu-latest
+          TARGET: manylinux
+        python-version: [3.7]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install twine wheel setuptools
+    - name: Build manylinux Python wheels
+      uses: RalfG/python-wheels-manylinux-build@v0.2.2-manylinux2010_x86_64
+      with:
+        python-versions: 'cp27-cp27mu cp35-cp35m cp36-cp36m cp37-cp37m cp38-cp38'
+        build-requirements: 'cython'
+        package-path: ''
+        pip-wheel-args: '--no-deps'

```

Comment:
Out of curiosity, what is the `--no-deps` doing?

Reply:
Per `pip wheel` documentation:
```
--no-deps
    Don’t install package dependencies.
```
I'm not sure if that's necessary for us. Thoughts?

Reply:
Does that apply to setting up the build environment, or is that somehow embedded into the resulting wheel?  Regardless, I think that you are correct and that it is not necessary (there are only a few Pyomo dependencies and they all install reliably).

Reply:
It's somehow embedded into the resulting wheel. Documentation for the [manylinux action](https://github.com/RalfG/python-wheels-manylinux-build) states that the default is `--no-deps` (hence why I didn't change it), but I'll remove that. 

(NOTE: Documentation also specifies that the `linux` wheels that are created NOT be uploaded to PyPi. Not sure why the Action creator doesn't just remove them in his action, but I can add a middle step to remove all `linux` files before we upload the artifacts, if you'd like.)

Reply:
@jsiirola - Ah, I remember why I had `--no-deps` now - my local testing fails because it can't find the right PyUtilib distribution (because PyUtilib 5.8.1.dev0 isn't on PyPi, doy). This isn't a problem if we do PyPi uploads first and THEN do the Pyomo wheel creation, however.